articles/openshift/oshift-how-portworx-snapshots.md

```diff
@@ -0,0 +1,347 @@
+
+    ```none
+    echo "apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      name: cloudsnap-restore
+      annotations:
+        snapshot.alpha.kubernetes.io/snapshot: test-cloudsnap
+    spec:
+      accessModes:
+         - ReadWriteOnce
+      storageClassName: stork-snapshot-sc
+      resources:
+        requests:
+          storage: 2Gi" | oc create -f -
+    ```
+
+## Creating scheduled snapshots
+
+You can create snapshot schedules that take recurring backups at a certain time each day, week, month or at a specified interval. For more information on each of the options see <https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/create-snapshots/scheduled/>.
+
+The following example creates a daily policy that creates snapshots at a certain time each day:
+
+1. Create some `SchedulePolicy` objects:
+
+    ```none
+    echo "apiVersion: stork.libopenstorage.org/v1alpha1
+    kind: SchedulePolicy
+    metadata:
+      name: daily
+    policy:
+      daily:
+        time: '10:00PM'
+        retain: 3" | oc create -f -
+    ```
+
+    ```none
+    echo "apiVersion: stork.libopenstorage.org/v1alpha1
+    kind: SchedulePolicy
+    metadata:
+      name: ten-minute
+    policy:
+      interval:
+        intervalMinutes: 10
+        retain: 3" | oc create -f -
+    ```
+
+2. Create a storage class that references one or more schedule policies. Any PVCs created from the storage class will have all schedule policies applied, For example, to backup the PVC locally every ten minutes and at 10:00PM daily to cloud-storage:
+
+    ```none
+    echo "kind: StorageClass
+    apiVersion: storage.k8s.io/v1
+    metadata:
+        name: test-schedule
+    provisioner: kubernetes.io/portworx-volume
+    parameters:
+       repl: '1'
+       snapshotschedule.stork.libopenstorage.org/default-schedule: |
+         schedulePolicyName: ten-minute
+         annotations:
+           portworx/snapshot-type: local
+       snapshotschedule.stork.libopenstorage.org/daily-schedule: |
+         schedulePolicyName: daily
+         annotations:
+           portworx/snapshot-type: cloud" | oc create -f -
+    ```
+
+3. Create a PVC from the storage class:
+
+    ```none
+    echo "apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      name: test-schedule
+    spec:
+      accessModes:
+         - ReadWriteOnce
+      storageClassName: test-schedule
+      resources:
+        requests:
+          storage: 2Gi" | oc create -f -
+    ```
+
+4. This should create some `volumeSnapshotSchedules`, for example:
+
+    ```none
+    oc get volumesnapshotschedules
+    
+    NAME                             AGE
+    test-schedule-daily-schedule     1m
+    test-schedule-default-schedule   1m
+    ```
+
+5. You can run a describe on these objects to ensure that they have created successfully and will be able to take snapshots.
+
```

Comment:
Can you actually show the results later after the policy has run successfully too.